### PR TITLE
[PSPE] Fireworks

### PR DESCRIPTION
**Pull request by 7kasper**
Fireworks in an image!

### CODE REVIEW

**Comment by 7kasper** [comment]
Whatever :S

File: src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityStatus.java [hunk 1]
```diff
@@ -21,10 +21,15 @@
...
 			NetworkEntity e = cache.getWatchedEntity(entityId);
-			//The only remap I know so far.
-			if((status == 31) && (e.getType() == NetworkEntityType.FISHING_FLOAT)) {
+			if ((status == 31) && (e.getType() == NetworkEntityType.FISHING_FLOAT)) {
 				status = 13;
 			}
+			if ((status == 10) && (e.getType() == NetworkEntityType.MINECART_TNT)) {
+				status = 31;
+			}
+			if ((status == 17) && (e.getType() == NetworkEntityType.FIREWORK)) {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityStatus.java]: I guess we will need a remap table for this soon

File: src/protocolsupport/zplatform/impl/glowstone/itemstack/GlowStoneNBTTagCompoundWrapper.java [hunk 2]
```diff
@@ -122,11 +122,16 @@ private Number getNumber(String key) {
 		}
 		return Integer.valueOf(0);
 	}
-
+
```
Comment (Shevchik) [inline src/protocolsupport/zplatform/impl/glowstone/itemstack/GlowStoneNBTTagCompoundWrapper.java]: No whitespaces pls

File: src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityStatus.java (hunk 1 above)
Comment (7kasper) [reply src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityStatus.java]: Yeah It's already a todo in there :S

File: src/protocolsupport/zplatform/impl/glowstone/itemstack/GlowStoneNBTTagCompoundWrapper.java (hunk 2 above)
Comment (7kasper) [reply src/protocolsupport/zplatform/impl/glowstone/itemstack/GlowStoneNBTTagCompoundWrapper.java]: Sure. Will commit in a moment.

**Pull request merged**